client/components/domains/register-domain-step/index.jsx

```diff
@@ -522,7 +521,6 @@ class RegisterDomainStep extends Component {
 					{ this.renderFilterContent() }
 					{ this.renderDomainExplanationImage() }
 					{ this.renderSideContent() }
-					<QueryContactDetailsCache />
```

Comment:
From what I can tell, this was never used. There are no selectors for the domain contact details in the register domain step.